### PR TITLE
Add Exfat support

### PR DESCRIPTION
**Pull request by russel5**
Like we already have kernel driver support and vold support

### CODE REVIEW

File: CommonConfig.mk [hunk 1]
```diff
@@ -42,6 +42,7 @@ TARGET_KERNEL_ARCH := arm64
 TARGET_KERNEL_HEADER_ARCH := arm64
 TARGET_KERNEL_CROSS_COMPILE_PREFIX := aarch64-linux-android-
 BOARD_KERNEL_IMAGE_NAME := Image.gz-dtb
+TARGET_EXFAT_DRIVER := sdfat
```
Comment (luk1337) [inline CommonConfig.mk]: maybe move it above TARGET_FS_CONFIG_GEN?

Comment (russel5) [reply CommonConfig.mk]: and what it change?

Comment (luk1337) [reply CommonConfig.mk]: it'll make me feel better, much better.

Comment (russel5) [reply CommonConfig.mk]: :)

**Comment by russel5** [comment]
@luk1337 now it's ok?

**Comment by jerpelea** [comment]
please push just the last commit the rest looks ok

**Pull request closed without merging**